tests/Cache/CacheClientTest.php

```diff
@@ -2119,10 +2119,6 @@ public function testDictionarySetFieldsWithNullItems_ThrowsException()
     public function testDictionarySetFieldsWithEmptyItems_IsError()
     {
         $dictionaryName = uniqid();
-        $items = [""];
```

Comment:
Unfortunately I don't know how to make this an invalid input while also allowing integer string keys.